stripe.go

```diff
@@ -485,3 +517,44 @@ func initUserAgent() {
 	}
 	encodedStripeUserAgent = string(marshaled)
 }
+
+// Checks if an error is a problem that we should retry on. This includes both
+// socket errors that may represent an intermittent problem and some special
+// HTTP statuses.
+func (s *BackendConfiguration) shouldRetry(err error, resp *http.Response, numRetries int) bool {
+	if numRetries >= s.MaxNetworkRetries {
+		return false
+	}
+
+	if err != nil {
+		return true
+	}
+
+	if resp.StatusCode == http.StatusConflict {
+		return true
+	}
+	return false
+}
+
+//sleepTime calculates sleeping/delay time in milliseconds between failure and a new one request.
+func (s *BackendConfiguration) sleepTime(numRetries int) (delay int) {
```

Comment:
Can you fix the spacing after the `//` here? And similar to the constants above, we should have this return a `time.Duration` instead of an integer. Lastly, is there a good reason to make this a named return value? I'd prefer to avoid those if possible because they're somewhat confusing unless you already know how they work.